frontend/src/components/AccountReceiveTableWarning.vue

```diff
@@ -11,10 +11,10 @@
     </q-card-section>
 
     <q-card-section>
-      <i18n path="AccountReceiveTableWarning.withdrawal-warning" tag="p">
+      <i18n-t keypath="AccountReceiveTableWarning.withdrawal-warning" tag="p">
```

Comment:
How does this fix work, i.e. (1) what is the difference between the `i18n` and `i18n-t` components, and (2) what is the `path` vs. `keypath` change? Are these just breaking changes from upgrading, or something different?

Reply:
Yes, `i18n-t` and `keypath` are breaking change renames for i18n v9 or later https://vue-i18n.intlify.dev/guide/migration/breaking.html#translation-component